screencaptor/src/main/java/com/wealthfront/screencaptor/ScreenCaptor.kt

```diff
@@ -69,8 +70,8 @@ object ScreenCaptor {
    */
   @Synchronized
   fun takeScreenshot(
```

Comment:
Can we keep the old api as well where you can pass in one view to take a screenshot of that? We can just wrap that view in a list and call this method.

Reply:
decided to keep the old View API, and add another that takes in an Activity instead of a list of View